manage.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+import os
+import sys
+
+if __name__ == "__main__":
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "tests.settings")
+    try:
+        from django.core.management import execute_from_command_line
+    except ImportError:
+        raise ImportError(
+            "Couldn't import Django. Are you sure it's installed and "
+            "available on your PYTHONPATH environment variable? Did you "
+            "forget to activate a virtual environment?"
+        )
```

Comment:
That's a good question. I think the reason stems from the fact that I actually pulled it from [flagsmith-ldap](https://github.com/Flagsmith/flagsmith-ldap/blob/main/manage.py) instead. The reason it's different _there_ (I think) is that flagsmith/flagsmith repo was created with django 2.x (because old) whereas flagsmith-ldap was created with django 3.x. I guess the actionable change here would be to update manage.py in the flagsmith/flagsmith repository rather than change anything here. 

A quick test shows my assumption is correct: 

```
❯ python -m django startproject testingmanagepy
❯ cd testingmanagepy
❯ cat manage.py
#!/usr/bin/env python
"""Django's command-line utility for administrative tasks."""
import os
import sys


def main():
    """Run administrative tasks."""
    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'testingmanagepy.settings')
    try:
        from django.core.management import execute_from_command_line
    except ImportError as exc:
        raise ImportError(
            "Couldn't import Django. Are you sure it's installed and "
            "available on your PYTHONPATH environment variable? Did you "
            "forget to activate a virtual environment?"
        ) from exc
    execute_from_command_line(sys.argv)


if __name__ == '__main__':
    main()
```

Reply:
Sounds good. Maybe make a ticket so we don't forget to update the other one?

Reply:
I just made the PR instead :) 

https://github.com/Flagsmith/flagsmith/pull/3403